models/reg_lstm/__main__.py

```diff
@@ -88,15 +86,20 @@ def evaluate_dataset(split_name, dataset_cls, model, embedding, loader, batch_si
     if args.dataset not in dataset_map:
         raise ValueError('Unrecognized dataset')
     else:
-        train_iter, dev_iter, test_iter = dataset_map[args.dataset].iters(args.data_dir, args.word_vectors_file, args.word_vectors_dir, batch_size=args.batch_size, device=args.gpu, unk_init=UnknownWordVecCache.unk)
+        train_iter, dev_iter, test_iter = dataset_map[args.dataset].iters(args.data_dir,
+                                                                          args.word_vectors_file,
+                                                                          args.word_vectors_dir,
+                                                                          batch_size=args.batch_size,
+                                                                          device=args.gpu,
+                                                                          unk_init=UnknownWordVecCache.unk)
 
     config = deepcopy(args)
     config.dataset = train_iter.dataset
     config.target_class = train_iter.dataset.NUM_CLASSES
     config.words_num = len(train_iter.dataset.TEXT_FIELD.vocab)
 
     print('Dataset {}    Mode {}'.format(args.dataset, args.mode))
-    print('VOCAB num',len(train_iter.dataset.TEXT_FIELD.vocab))
+    print('VOCAB num', len(train_iter.dataset.TEXT_FIELD.vocab))
```

Comment:
VOCAB?

Reply:
This part is sketchy in all of the main methods. I guess this is why it's not a great idea to have half of the methods in the code base copy-pasted huh~